### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-server#128]
https://app.getsentry.com/gns3/gns3-server/group/60458022/

```
Stacktrace (appel le plus récent en dernier) : 

  File "c:\users\gns3\documents\gns3 build for windows\gns3-server\gns3server\web\route.py", line 139, in control_schema
  File "c:\users\gns3\documents\gns3 build for windows\gns3-server\gns3server\handlers\api\virtualbox_handler.py", line 191, in start
  File "c:\users\gns3\documents\gns3 build for windows\gns3-server\gns3server\modules\virtualbox\virtualbox_vm.py", line 177, in start
  File "c:\users\gns3\documents\gns3 build for windows\gns3-server\gns3server\modules\virtualbox\virtualbox_vm.py", line 647, in _set_network_options
```

**Comment by noplay** [comment]
I think  it's related to
https://github.com/GNS3/gns3-gui/issues/255

**Comment by noplay** [comment]
Fixed via #135
